dataland-frontend/tests/e2e/specs/admin-tools/AdminConsoleSecurity.ts

```diff
@@ -0,0 +1,31 @@
+describe("As a user I expect the admin console only to be reachable using admin-proxy and not from remote", () => {
+  function checkThatUrlResolvesToErrorPage(url: string) {
+    cy.visit(url);
+    cy.get("h2").should("exist").should("contain", "Sorry an error occurred!");
+    cy.url().should("contain", "nocontent");
+  }
+
+  it(`Test Admin Console not reachable from remote`, () => {
+    checkThatUrlResolvesToErrorPage("/keycloak/admin");
+  });
+
+  it(`Master Realm not reachable from remote`, () => {
+    checkThatUrlResolvesToErrorPage("/keycloak/realms/master");
+  });
+
+  it(`Test Admin Console is reachable via dataland-admin`, () => {
```

Comment:
Dieser Test ist doch bestimmt so geschaltet, dass er nur in der cypress execution environment "developmentLocal" läuft, oder?